### PR TITLE
feat(edition): reference Access Edition where applicable

### PR DESCRIPTION
**Pull request by educhastenier**
Relates to https://bonitasoft.atlassian.net/browse/DEV-612

### CODE REVIEW

File: modules/identity/pages/profile-creation.adoc
Comment (educhastenier) [inline modules/identity/pages/profile-creation.adoc:57]: This sentence was already not clear. Who knows what is the truth?

File: modules/runtime/pages/set-log-and-archive-levels.adoc
Comment (educhastenier) [inline modules/runtime/pages/set-log-and-archive-levels.adoc]: I would remove this here, to avoid double maintenance

File: modules/runtime/pages/tomcat-bundle.adoc
Comment (educhastenier) [inline modules/runtime/pages/tomcat-bundle.adoc:58]: As a consequence, we should remove them from the distrib and make them available somewhere else:
* ldap-synch: already done
* tools/SSO/cas: to do
* tools/SSO/kerberos: to do

Comment (abirembaut) [reply modules/runtime/pages/tomcat-bundle.adoc:58]: I reverted tools/CAS removal for the moment since it is still there

File: modules/runtime/pages/tomcat-bundle.adoc
Comment (educhastenier) [inline modules/runtime/pages/tomcat-bundle.adoc]:
```suggestion
* `setup/`: a tool to manage Bonita Platform configuration, stored in database instead of filesystem. Also ships a tool to centralize all the required Tomcat bundle configuration.
* `tools/request_key_utils`: folder containing the script to generate license request keys (Subscription editions only).
```

File: modules/runtime/pages/set-log-and-archive-levels.adoc
Comment (abirembaut) [reply modules/runtime/pages/set-log-and-archive-levels.adoc]: all the line ?

Comment (educhastenier) [reply modules/runtime/pages/set-log-and-archive-levels.adoc]: By default, all the information is archived but in some editions this is xref:ROOT:configurable-archive.adoc[configurable].

File: modules/identity/pages/organization-management-in-bonita-bpm-studio.adoc
Comment (abirembaut) [inline modules/identity/pages/organization-management-in-bonita-bpm-studio.adoc]:
```suggestion
*_In Teamwork edition_*, the 2 provided profiles above can be edited in the Studio using an XML editor. For development purposes, all users mapped to role _member_ (cf _ACME_ organization) will benefit from those 2 profiles in the Bonita Applications to let you log in with any of those users and test your processes.
```
minor update and removal of the last sentence that can be confusing

File: modules/runtime/pages/monitoring.adoc
Comment (abirembaut) [inline modules/runtime/pages/monitoring.adoc:12]: Not in Efficiency according to Support

**Review by abirembaut — CHANGES_REQUESTED** [review]
LGTM
just 2 comments

File: modules/runtime/pages/monitoring.adoc
Comment (educhastenier) [reply modules/runtime/pages/monitoring.adoc:12]: Ok, I let you modify it directly

Comment (abirembaut) [reply modules/runtime/pages/monitoring.adoc:12]: my mistake. this page is for process monitoring. so it is ok

**Review by abirembaut — APPROVED** [review]

**Comment by pabloalonso** [comment]
WE DO NOT MERGE THIS PR without @pabloalonso approval

**Comment by rbioteau** [comment]
Why can't we merge this pr now ?

**Comment by educhastenier** [comment]
> WE DO NOT MERGE THIS PR without @pabloalonso approval

@pabloalonso update?

**Pull request merged**
